modules/RTC/TraceablePeerConnection.js

```diff
@@ -406,48 +406,49 @@ function extractSSRCMap(desc) {
     if (typeof desc !== 'object' || desc === null ||
         typeof desc.sdp !== 'string') {
         logger.warn('An empty description was passed as an argument.');
-        return desc;
+        return;
```

Comment:
This method was inconsistent already, I merely made it return the same thing in case of failure. There's no JSDoc so who knows what should it return.